src/containerapp/azext_containerapp/custom.py

```diff
@@ -3844,21 +3844,17 @@ def show_environment_premium_ingress(cmd, name, resource_group_name):
         handle_raw_exception(e)
 
 
-def add_environment_premium_ingress(cmd, name, resource_group_name, workload_profile_name, min_replicas, max_replicas, termination_grace_period=None, request_idle_timeout=None, header_count_limit=None, no_wait=False):
+def add_environment_premium_ingress(cmd, name, resource_group_name, workload_profile_name, min_replicas=None, max_replicas=None, termination_grace_period=None, request_idle_timeout=None, header_count_limit=None, no_wait=False):
     _validate_subscription_registered(cmd, CONTAINER_APPS_RP)
 
     try:
         ManagedEnvironmentPreviewClient.show(cmd, resource_group_name, name)
         env_patch = {}
         ingress_config = {}
         safe_set(env_patch, "properties", "ingressConfiguration", value=ingress_config)
-        scale = {}
```

Comment:
pls keep the code until parameter is removed. Currently they are in deprecated mode and hidden to customers.

Reply:
The control plane change has already fully deployed, the properties are ignored and there's no reason to send them.